main.py

```diff
@@ -1,10 +1,59 @@
-from kivymd.app import MDApp
-from kivymd.uix.label import MDLabel
+import tkinter as tk
+from turtle import clear
+import markdown as md
+from tkhtmlview import HTMLLabel
 
 
-class MainApp(MDApp):
-    def build(self):
-        return MDLabel(text="Hello, World", halign="center")
+def showHTML():
+    # remove show button
+    show_button.pack_forget()
+    text = text_entry.get("1.0", "end")
+    html = md.markdown(text)
+    global html_area
+    global clear_button
+    html_area = HTMLLabel(window, html=html)
+    html_area.pack()
+    clear_button = tk.Button(window, text="Clear HTML", command=clearHTML)
+    clear_button.pack(side=tk.BOTTOM)
 
 
-MainApp().run()
\ No newline at end of file
+def clearHTML():
+    try:
+        html_area.pack_forget()
+        clear_button.pack_forget()
+        show_button.pack(side=tk.BOTTOM)
+    except:
+        pass
+
+
+# base for the GUI window
```

Comment:
We should encapsulate this in some sort of initialization function and a main function so we don't have a bunch of globals